### ISSUE TITLE
HEIF Not Available in vips Command After Importing ruby-vips

### ISSUE THREAD

**Issue by psfrolov** [libvips/ruby-vips#311]
**Environment**

OS: MacOS Big Sur 11.4
Ruby: ruby 3.0.1p64 (2021-04-05 revision 0fb782ee38) [x86_64-darwin20]
ruby-vips 2.1.2
vips: stable 8.11.0 

**To Reproduce**
1. `brew install vips`
2. `gem install ruby-vips`
3. `irb`
    1. ``STDOUT.puts `vips -l filesave` ``
    Output:
    ```
     VipsForeignSave (filesave), file savers, priority=0, mono
        VipsForeignSaveCsv (csvsave_base), save image to csv (.csv), priority=0, mono
          VipsForeignSaveCsvFile (csvsave), save image to csv (.csv), priority=0, mono
          VipsForeignSaveCsvTarget (csvsave_target), save image to csv (.csv), priority=0, mono
        VipsForeignSaveMatrix (matrixsave_base), save image to matrix (.mat), priority=0, mono
          VipsForeignSaveMatrixFile (matrixsave), save image to matrix (.mat), priority=0, mono
          VipsForeignSaveMatrixTarget (matrixsave_target), save image to matrix (.mat), priority=0, mono
          VipsForeignPrintMatrix (matrixprint), print matrix (.mat), priority=0, mono
        VipsForeignSaveRaw (rawsave), save image to raw file (.raw), priority=0, any
        VipsForeignSaveRawFd (rawsave_fd), write raw image to file descriptor (.raw), priority=0, any
        VipsForeignSaveVips (vipssave_base), save vips base class (.v, .vips), priority=0, any
          VipsForeignSaveVipsFile (vipssave), save image to file in vips format (.v, .vips), priority=0, any
          VipsForeignSaveVipsTarget (vipssave_target), save image to target in vips format (.v, .vips), priority=0, any
        VipsForeignSavePpm (ppmsave_base), save to ppm (.ppm, .pgm, .pbm, .pfm), priority=0, rgb
          VipsForeignSavePpmFile (ppmsave), save image to ppm file (.ppm, .pgm, .pbm, .pfm), priority=0, rgb
          VipsForeignSavePpmTarget (ppmsave_target), save to ppm (.ppm, .pgm, .pbm, .pfm), priority=0, rgb
        VipsForeignSaveRad (radsave_base), save Radiance (.hdr), priority=0, rgb
          VipsForeignSaveRadFile (radsave), save image to Radiance file (.hdr), priority=0, rgb
          VipsForeignSaveRadBuffer (radsave_buffer), save image to Radiance buffer (.hdr), priority=0, rgb
          VipsForeignSaveRadTarget (radsave_target), save image to Radiance target (.hdr), priority=0, rgb
        VipsForeignSaveJp2k (jp2ksave_base), save image in JPEG2000 format (.j2k, .jp2, .jpt, .j2c, .jpc), priority=0, any
          VipsForeignSaveJp2kFile (jp2ksave), save image in JPEG2000 format (.j2k, .jp2, .jpt, .j2c, .jpc), priority=0, any
          VipsForeignSaveJp2kBuffer (jp2ksave_buffer), save image in JPEG2000 format (.j2k, .jp2, .jpt, .j2c, .jpc), priority=0, any
          VipsForeignSaveJp2kTarget (jp2ksave_target), save image in JPEG2000 format (.j2k, .jp2, .jpt, .j2c, .jpc), priority=0, any
        VipsForeignSaveDz (dzsave_base), save image to deep zoom format (.dz), priority=0, any
          VipsForeignSaveDzFile (dzsave), save image to deepzoom file (.dz), priority=0, any
          VipsForeignSaveDzBuffer (dzsave_buffer), save image to dz buffer (.dz), priority=0, any
        VipsForeignSavePng (pngsave_base), save png (.png), priority=0, rgba
          VipsForeignSavePngFile (pngsave), save image to png file (.png), priority=0, rgba
          VipsForeignSavePngBuffer (pngsave_buffer), save image to png buffer (.png), priority=0, rgba
          VipsForeignSavePngTarget (pngsave_target), save image to target as PNG (.png), priority=0, rgba
        VipsForeignSaveJpeg (jpegsave_base), save jpeg (.jpg, .jpeg, .jpe), priority=0, rgb-cmyk
          VipsForeignSaveJpegFile (jpegsave), save image to jpeg file (.jpg, .jpeg, .jpe), priority=0, rgb-cmyk
          VipsForeignSaveJpegBuffer (jpegsave_buffer), save image to jpeg buffer (.jpg, .jpeg, .jpe), priority=0, rgb-cmyk
          VipsForeignSaveJpegTarget (jpegsave_target), save image to jpeg target (.jpg, .jpeg, .jpe), priority=0, rgb-cmyk
          VipsForeignSaveJpegMime (jpegsave_mime), save image to jpeg mime (.jpg, .jpeg, .jpe), priority=0, rgb-cmyk
        VipsForeignSaveWebp (webpsave_base), save webp (.webp), priority=0, rgba-only
          VipsForeignSaveWebpFile (webpsave), save image to webp file (.webp), priority=0, rgba-only
          VipsForeignSaveWebpBuffer (webpsave_buffer), save image to webp buffer (.webp), priority=0, rgba-only
          VipsForeignSaveWebpTarget (webpsave_target), save image to webp target (.webp), priority=0, rgba-only
        VipsForeignSaveTiff (tiffsave_base), save image to tiff file (.tif, .tiff), priority=0, any
          VipsForeignSaveTiffFile (tiffsave), save image to tiff file (.tif, .tiff), priority=0, any
          VipsForeignSaveTiffBuffer (tiffsave_buffer), save image to tiff buffer (.tif, .tiff), priority=0, any
        VipsForeignSaveMagick (magicksave_base), save with ImageMagick (.gif, .bmp), priority=-100, any
          VipsForeignSaveMagickFile (magicksave), save file with ImageMagick (.gif, .bmp), priority=-100, any
          VipsForeignSaveMagickBuffer (magicksave_buffer), save image to magick buffer (.gif, .bmp), priority=-100, any
        VipsForeignSaveFits (fitssave), save image to fits file (.fits, .fit, .fts), priority=0, any
        VipsForeignSaveHeif (heifsave_base), save image in HEIF format (.heic, .heif, .avif), priority=0, rgba-only
          VipsForeignSaveHeifFile (heifsave), save image in HEIF format (.heic, .heif, .avif), priority=0, rgba-only
          VipsForeignSaveHeifBuffer (heifsave_buffer), save image in HEIF format (.heic, .heif, .avif), priority=0, rgba-only
          VipsForeignSaveHeifTarget (heifsave_target), save image in HEIF format (.heic, .heif, .avif), priority=0, rgba-only
    ``` 
    2. `require 'ruby-vips'`
    3. ``STDOUT.puts `vips -l filesave` ``
    Output:
    ```
       VipsForeignSaveCsv (csvsave_base), save image to csv (.csv), priority=0, mono
          VipsForeignSaveCsvFile (csvsave), save image to csv (.csv), priority=0, mono
          VipsForeignSaveCsvTarget (csvsave_target), save image to csv (.csv), priority=0, mono
        VipsForeignSaveMatrix (matrixsave_base), save image to matrix (.mat), priority=0, mono
          VipsForeignSaveMatrixFile (matrixsave), save image to matrix (.mat), priority=0, mono
          VipsForeignSaveMatrixTarget (matrixsave_target), save image to matrix (.mat), priority=0, mono
          VipsForeignPrintMatrix (matrixprint), print matrix (.mat), priority=0, mono
        VipsForeignSaveRaw (rawsave), save image to raw file (.raw), priority=0, any
        VipsForeignSaveRawFd (rawsave_fd), write raw image to file descriptor (.raw), priority=0, any
        VipsForeignSaveVips (vipssave_base), save vips base class (.v, .vips), priority=0, any
          VipsForeignSaveVipsFile (vipssave), save image to file in vips format (.v, .vips), priority=0, any
          VipsForeignSaveVipsTarget (vipssave_target), save image to target in vips format (.v, .vips), priority=0, any
        VipsForeignSavePpm (ppmsave_base), save to ppm (.ppm, .pgm, .pbm, .pfm), priority=0, rgb
          VipsForeignSavePpmFile (ppmsave), save image to ppm file (.ppm, .pgm, .pbm, .pfm), priority=0, rgb
          VipsForeignSavePpmTarget (ppmsave_target), save to ppm (.ppm, .pgm, .pbm, .pfm), priority=0, rgb
        VipsForeignSaveRad (radsave_base), save Radiance (.hdr), priority=0, rgb
          VipsForeignSaveRadFile (radsave), save image to Radiance file (.hdr), priority=0, rgb
          VipsForeignSaveRadBuffer (radsave_buffer), save image to Radiance buffer (.hdr), priority=0, rgb
          VipsForeignSaveRadTarget (radsave_target), save image to Radiance target (.hdr), priority=0, rgb
        VipsForeignSaveJp2k (jp2ksave_base), save image in JPEG2000 format (.j2k, .jp2, .jpt, .j2c, .jpc), priority=0, any
          VipsForeignSaveJp2kFile (jp2ksave), save image in JPEG2000 format (.j2k, .jp2, .jpt, .j2c, .jpc), priority=0, any
          VipsForeignSaveJp2kBuffer (jp2ksave_buffer), save image in JPEG2000 format (.j2k, .jp2, .jpt, .j2c, .jpc), priority=0, any
          VipsForeignSaveJp2kTarget (jp2ksave_target), save image in JPEG2000 format (.j2k, .jp2, .jpt, .j2c, .jpc), priority=0, any
        VipsForeignSaveDz (dzsave_base), save image to deep zoom format (.dz), priority=0, any
          VipsForeignSaveDzFile (dzsave), save image to deepzoom file (.dz), priority=0, any
          VipsForeignSaveDzBuffer (dzsave_buffer), save image to dz buffer (.dz), priority=0, any
        VipsForeignSavePng (pngsave_base), save png (.png), priority=0, rgba
          VipsForeignSavePngFile (pngsave), save image to png file (.png), priority=0, rgba
          VipsForeignSavePngBuffer (pngsave_buffer), save image to png buffer (.png), priority=0, rgba
          VipsForeignSavePngTarget (pngsave_target), save image to target as PNG (.png), priority=0, rgba
        VipsForeignSaveJpeg (jpegsave_base), save jpeg (.jpg, .jpeg, .jpe), priority=0, rgb-cmyk
          VipsForeignSaveJpegFile (jpegsave), save image to jpeg file (.jpg, .jpeg, .jpe), priority=0, rgb-cmyk
          VipsForeignSaveJpegBuffer (jpegsave_buffer), save image to jpeg buffer (.jpg, .jpeg, .jpe), priority=0, rgb-cmyk
          VipsForeignSaveJpegTarget (jpegsave_target), save image to jpeg target (.jpg, .jpeg, .jpe), priority=0, rgb-cmyk
          VipsForeignSaveJpegMime (jpegsave_mime), save image to jpeg mime (.jpg, .jpeg, .jpe), priority=0, rgb-cmyk
        VipsForeignSaveWebp (webpsave_base), save webp (.webp), priority=0, rgba-only
          VipsForeignSaveWebpFile (webpsave), save image to webp file (.webp), priority=0, rgba-only
          VipsForeignSaveWebpBuffer (webpsave_buffer), save image to webp buffer (.webp), priority=0, rgba-only
          VipsForeignSaveWebpTarget (webpsave_target), save image to webp target (.webp), priority=0, rgba-only
        VipsForeignSaveTiff (tiffsave_base), save image to tiff file (.tif, .tiff), priority=0, any
          VipsForeignSaveTiffFile (tiffsave), save image to tiff file (.tif, .tiff), priority=0, any
          VipsForeignSaveTiffBuffer (tiffsave_buffer), save image to tiff buffer (.tif, .tiff), priority=0, any
        VipsForeignSaveMagick (magicksave_base), save with ImageMagick (.gif, .bmp), priority=-100, any
          VipsForeignSaveMagickFile (magicksave), save file with ImageMagick (.gif, .bmp), priority=-100, any
          VipsForeignSaveMagickBuffer (magicksave_buffer), save image to magick buffer (.gif, .bmp), priority=-100, any
        VipsForeignSaveFits (fitssave), save image to fits file (.fits, .fit, .fts), priority=0, any
    ``` 
In the second output (after importing ruby-vips) there is no `VipsForeignSaveHeif` at the bottom.

**Comment by jcupitt** [comment]
Hi, this was fixed in 8.11.1, hopefully homebrew will update soon.

**Comment by psfrolov** [comment]
vips 8.11.1 is landed in homebrew today -- fixed!
